examples/plot_8_animations.py

```diff
@@ -3,6 +3,9 @@
 ============================
 
 Show a Matplotlib animation, which should end up nicely embedded below.
+
+In order to enable support for animations ``'matplotlib_animations'``
+must be set to ``True`` in the sphinx gallery configuration.
```

Comment:
Let's add a nice link, too!
```suggestion
must be set to ``True`` in the sphinx gallery
:ref:`configuration <image_scrapers>`.
```